arduino/cores/packagemanager/install_uninstall_test.go

```diff
@@ -0,0 +1,170 @@
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino Diecimila"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino Nano"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino/Genuino Mega"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino MegaADK"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino Leonardo"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino Leonardo Ethernet"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino/Genuino Micro"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino Esplora"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino Mini"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino Ethernet"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino Fio"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino BT"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino LilyPadUSB"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino Lilypad"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino Pro"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino ATMegaNG"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino Robot Control"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino Robot Motor"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino Gemma"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Adafruit Circuit Playground"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino Yún Mini"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Arduino Industrial 101"})
+	require.Contains(t, indexPlatformRelease.Boards, packageindex.IndexBoard{Name: "Linino One"})
+	require.Contains(t, indexPlatformRelease.ToolDependencies,
+		packageindex.IndexToolDependency{
+			Packager: "arduino",
+			Name:     "avr-gcc",
+			Version:  semver.ParseRelaxed("5.4.0-atmel3.6.1-arduino2"),
+		})
+	require.Contains(t, indexPlatformRelease.ToolDependencies,
+		packageindex.IndexToolDependency{
+			Packager: "arduino",
+			Name:     "avrdude",
+			Version:  semver.ParseRelaxed("6.3.0-arduino14"),
+		})
+	require.Contains(t, indexPlatformRelease.ToolDependencies,
+		packageindex.IndexToolDependency{
+			Packager: "arduino",
+			Name:     "arduinoOTA",
+			Version:  semver.ParseRelaxed("1.2.1"),
+		})
```

Comment:
It's ok-ish doing this way but, there must be a better way to check for matching JSONs :-)